src/standalone/Vegetation/canopy_parameterizations.jl

```diff
@@ -24,71 +24,164 @@ export plant_absorbed_ppfd,
+(APAR, ANIR).
+"""
+function compute_absorbances(
+    RT::BeerLambertModel{FT},
+    PAR,
+    NIR,
+    LAI,
+    K,
+    _,
+    _,
+    _,
+) where {FT}
+    RTP = RT.parameters
+    APAR = @. plant_absorbed_pfd(RT, PAR, RTP.α_PAR_leaf, LAI, K)
+    ANIR = @. plant_absorbed_pfd(RT, NIR, RTP.α_NIR_leaf, LAI, K)
+    return (APAR, ANIR)
+end
+
+"""
+    compute_absorbances(
+        RT::TwoStreamModel{FT},
+        PAR,
+        NIR,
+        LAI,
+        K,
+        θs,
+        α_soil_PAR,
+        α_soil_NIR,
+    )
 
-This applies the Beer-Lambert law, which is a function of photosynthetically
-active radiation (`PAR`; moles of photons/m^2/),
-PAR canopy reflectance (`ρ_leaf`), the extinction
-coefficient (`K`), leaf area index (`LAI`) and the clumping index (`Ω`). 
+Compute APAR and ANIR absorbances for a canopy in the case of the
+two-stream model. The absorbances are a function of the radiative transfer 
+model, as well as the magnitude of incident PAR and NIR radiation in moles of 
+photons, the leaf areaindex, the extinction coefficient, the solar zenith angle,
+and soil albedo. Returns a tuple of (APAR, ANIR).
+"""
+function compute_absorbances(
+    RT::TwoStreamModel{FT},
+    PAR,
+    NIR,
+    LAI,
+    K,
+    θs,
+    α_soil_PAR,
+    α_soil_NIR,
+) where {FT}
+    RTP = RT.parameters
+    APAR = @. plant_absorbed_pfd(
+        RT,
+        PAR,
+        RTP.α_PAR_leaf,
+        RTP.τ_PAR_leaf,
+        LAI,
+        K,
+        θs,
+        α_soil_PAR,
+    )
+    ANIR = @. plant_absorbed_pfd(
+        RT,
+        NIR,
+        RTP.α_NIR_leaf,
+        RTP.τ_NIR_leaf,
+        LAI,
+        K,
+        θs,
+        α_soil_NIR,
+    )
+    return (APAR, ANIR)
+end
+
+"""
+    plant_absorbed_pfd(
+        RT::BeerLambertModel{FT},
+        SW_IN:FT,
+        α_leaf::FT,
+        LAI::FT,
+        K::FT,
+    )
+
+Computes the absorbed photon flux density in terms of mol photons per m^2 per 
+second for a radiation band. If the reflectance and radiation for NIR is passed, 
+computes ANIR and if PAR reflectance and rediation are passed, computes APAR.
+
+This applies the Beer-Lambert law, which is a function of incident 
+radiation (`SW_IN`; moles of photons/m^2/), leaf reflectance
+(`α_PAR_leaf`), the extinction coefficient (`K`), leaf area index (`LAI`),
+and the clumping index (`Ω`). 
 The function takes in all parameters in the parameters struct for a 
-BeerLambertModel, along with the PAR, LAI, extinction coefficient K, and solar 
-zenith anlgle.
+BeerLambertModel, along with the SW_IN, LAI, extinction coefficient K, and solar 
+zenith angle.
 """
-function plant_absorbed_ppfd(
+function plant_absorbed_pfd(
     RT::BeerLambertModel{FT},
-    PAR::FT,
+    SW_IN::FT,
+    α_leaf::FT,
     LAI::FT,
     K::FT,
-    _,
-    _,
 ) where {FT}
     RTP = RT.parameters
-    APAR = PAR * (1 - RTP.ρ_leaf) * (1 - exp(-K * LAI * RTP.Ω))
-    return APAR
+    AR = SW_IN * (1 - α_leaf) * (1 - exp(-K * LAI * RTP.Ω))
```

Comment:
Not a big deal but for consistency, we could agree to use either "radiation" (W m-2) or "photon flux density" (umol m-2 s-1) throughout. 

Here we call the function "PFD" but return "AR".

It sounds like we could use "radiation" everywhere, since the term PAR and APAR are more common. We should in theory therefore stick to (W m-2) units, but we could do like everyone in ecology and wrongfully attribute PAR units of (umol m-2 s-1).